python/src/LinearModelTest_doc.i.in

```diff
@@ -354,3 +354,125 @@ Examples
+secondSample : 2-d sequence of float
+    Second tested sample, of dimension 1.
+selection : sequence of int, maximum integer value :math:`< n`
+    List of indices selecting which subsets of the first sample will successively
+    be tested with the second sample through the regression test.
+level : positive float :math:`< 1`
+    Threshold p-value of the test (= first kind risk), it must be
+    :math:`< 1`, equal to 0.05 by default.
+
+Returns
+-------
+testResults : Collection of :class:`~openturns.TestResult`
+    Results for each component of the linear model including intercept.
+
+See Also
+--------
+LinearModelTest_FullRegression, LinearModelTest_LinearModelFisher
+
+Notes
+-----
+The Partial Regression Test is used to check the quality of the linear regression
+AnalyticalFmodel between two samples: *firstSample* of dimension *n* and *secondSample* of
+dimension 1. The parameter *selection* enables to select specific subsets of the
+*firstSample* to be tested.
+
+Examples
+--------
+>>> import openturns as ot
+>>> ot.RandomGenerator.SetSeed(0)
+>>> dim = 3
+>>> distCol = [ot.Normal()] * dim
+>>> S = ot.CorrelationMatrix(dim)
+>>> S[0, dim - 1] = 0.99
+>>> copula = ot.NormalCopula(S)
+>>> distribution = ot.ComposedDistribution(distCol, copula)
+>>> sample = distribution.getSample(30)
+>>> firstSample = sample[:, :2]
+>>> secondSample = sample[:, 2]
+>>> selection = [1]
+>>> test_result = ot.LinearModelTest.PartialRegression(firstSample, secondSample, selection)
+>>> print(test_result[1])
+class=TestResult name=Unnamed type=Regression binaryQualityMeasure=true p-value threshold=0.05 p-value=0.579638 description=[]"
+
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LinearModelTest::FullRegression
+"Test whether two discrete samples are not linear.
+
+**Available usages**:
+
+    LinearModelTest.FullRegression(*firstSample, secondSample*)
+
+    LinearModelTest.FullRegression(*firstSample, secondSample, level*)
+
+
+Parameters
+----------
+firstSample : 2-d sequence of float
+    First tested sample, of dimension :math:`n \geq 1`.
+secondSample : 2-d sequence of float
+    Second tested sample, of dimension 1.
+level : positive float :math:`< 1`
+    Threshold p-value of the test (= first kind risk), it must be
+    :math:`< 1`, equal to 0.05 by default.
+
+Returns
+-------
+testResults : Collection of :class:`~openturns.TestResult`
+    Results for each component of the linear model including intercept.
+
+See Also
+--------
+LinearModelTest_PartialRegression, LinearModelTest_LinearModelFisher
+
+Notes
+-----
+The Full Regression Test is used to check the quality of the linear regression
+model between two samples: *firstSample* of dimension *n* and *secondSample* of
+dimension 1. If *firstSample[i]* is the sample extracted from
+*firstSample* (:math:`i^{th}` coordinate of each point of the sample),
+FullRegression performs the linear regression test simultaneously on all
```

Comment:
not simultaneously, it is not the Fisher test

Reply:
ok